sandbox-templates/react-markdown-with-vite/src/main.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+// eslint-disable-next-line n/file-extension-in-import
+import ReactDOM from 'react-dom/client'
```

Comment:
this lint rule will break the code by adding a `.js` extension.
`react-dom/client` is in the export map, `react-dom/client.js` is not a thing.